tests/fixtures/assetsGroups.ts

```diff
@@ -0,0 +1,34 @@
+import { AssetsGroupsBody } from "../../lib/modules/asset/types/AssetGroupContent";
+
+export const assetGroupTestId = "test-group";
```

Comment:
It's really nitpicking but IMHO the example would be more explicit with "real" hierarchic groups like Occitanie, Montpellier, Ecusson, Rue des écoles Pies